### ISSUE TITLE
blkid zfs support: supposed to work?

### ISSUE THREAD

**Issue by RJVB** [openzfsonosx/zfs#685]
Is the test for blkid ZFS support supposed to fail for other reason than that it assumes the GNU mktemp and truncate utilities?

https://github.com/openzfsonosx/zfs/blob/d48d8749f240018c11a197bc083ff43efa9b2d3b/config/user-libblkid.m4#L38

**Comment by lundman** [comment]
Not really, we inherited it from ZOL over time - but it is a Linux thing afaik.
